.github/workflows/check-plugin.yml

```diff
@@ -1,8 +1,8 @@
 name: Check Plugin Availability Main
 
 on:
-  #schedule:
-  #  - cron: '0 0,15 * * *'
```

Comment:
You delete an extra space, please make sure indentation is always 2 spaces

Reply:
Resolved